tests/pkcli/build_data/1.out/srv/host/v9.radia.run/srv/mlflow/env

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
not sure this file is necessary unless you are testing the env var.

Reply:
I think it is worth testing in this case. If the env var is missing the server will start fine just be broken. It will use the default user/pass which seems like a large enough security risk it is worth trying to catch.